components/LogoNav/LogoNav.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import LogoNav from './LogoNav';
+
+describe('LogoNav Component', () => {
```

Comment:
The testing looks great! You are testing that the logo renders and that there is a link with a certain route attached to it. But, don't you need to mock a router.push that goes to /league/all when the link is clicked?